client/src/components/Ticketing/ticketingmanager/Season/components/SeasonInfo.tsx

```diff
@@ -9,11 +9,12 @@ import {
   deleteSeasonInfo,
   updateEventSeason,
 } from './utils/apiRequest';
-import {seasonDefaultValues, SeasonProps} from './utils/seasonCommon';
+import {seasonDefaultValues, SeasonProps, SeasonTicketValues} from './utils/seasonCommon';
 import ViewSeasonInfo from './utils/ViewSeasonInfo';
 import {LoadingScreen} from '../../../mainpage/LoadingScreen';
+import {SeasonTicketTypeUpdateTable} from './utils/SeasonTicketTypeUpdateTable';
 
-const SeasonInfo = (props: SeasonProps) => {
+const SeasonInfo = (props: SeasonProps& { onUpdateSeasonTicketType: (requestData: any) => Promise<void> }) => {
```

Comment:
Yes, it's the syntax for intersection of `SeasonProps` and `onUpdateSeasonTicketType` from `seasonContainer`, since the data must be sent back in the same form.

Reply:
Oh interesting